Makefile

```diff
@@ -4,6 +4,10 @@ SOURCES := src src/common src/desktop
 ARGS    := -Ofast -flto -std=c++11 #-DDEBUG
 LIBS    := -lportaudio
 
+APPNAME      ?= NooDS
+ORGANIZATION ?= com.github.hydr8gon
+DESTDIR      ?= /usr/local

```

Comment:
Aren't things normally installed to `/usr`? `/usr/local` is a barren wasteland on my machine.